tests/golden/eval_output

```diff
@@ -2,9 +2,9 @@
 Sentence-Level Stats:
 	correct:  1636
 	total:  2200
-	% correct:  74.3636363636
+	% correct:  74.36363636363636
```

Comment:
Instead of changing this, could we just change the print call so that it preserves the legacy behavior? I just want to make it super obvious to anyone reading that we didn't change behavior in this PR.